flaskr/models.py

```diff
@@ -1,8 +1,14 @@
 from datetime import datetime
-from flaskr import db
+from flaskr import db, login_manager
+from flask_login import UserMixin
 
 
-class User(db.Model):
+@login_manager.user_loader
+def load_user(user_id):
+    return User.query.get(int(user_id))
+
+
+class User(db.Model, UserMixin):
     id = db.Column(db.Integer, primary_key=True)
     username = db.Column(db.String(20), unique=True, nullable=False)
     email = db.Column(db.String(120), unique=True, nullable=False)
```

Comment:
UserMixin which provides default implementation of certain attributes & methods on User model
i.e is_authenticated, is_active, is_anonymous, get_id()